ci/build_container/build_recipes/boringssl.sh

```diff
@@ -1,10 +1,16 @@
+#!/bin/bash
```

Comment:
Scripts that are not scripts are just annoying... and might end up in weird issues down the road.

Granted, this change is not particularly useful, but it doesn't hurt either.